spec/system/admin/reports/packing_report_spec.rb

```diff
@@ -32,9 +32,12 @@
     let(:variant2) { create(:variant, product: product1, unit_description: "Small") }
     let(:product2) { create(:simple_product, name: "Product 2", supplier: supplier) }
 
+    let(:open_datetime) { (Time.zone.now - 1.month).strftime("%Y-%m-%d 00:00") }
```

Comment:
This will work well until the spec is run at midnight. I know, it's unlikely but it does happen. The best solution I can think of is to add an `around` block to wrap the whole thing in `Timecop.freeze`.

Reply:
Ahh, true - good point. I've tried to improve it in this commit ce96094f4923a4b4caa2c4556d81312eacea1890.